scripts/update-endpoints.js

```diff
@@ -3,9 +3,13 @@ const fs = require('fs');
 const url = 'https://chains.hyperquery.xyz/active_chains';
 const renameConfig = {
   eth: 'Ethereum Mainnet',
+  "polygon-zkevm": "Polygon zkEVM",
+  "zksync": "ZKsync",
   // Add other renaming rules here
 };
 
+const filterEndpoints = [/^staging-/];
```

Comment:
This regular expression is used to filter out any chain starting with `staging-` (such as staging-eth